dts/sdm450-samsung-r05.dts

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "msm8953.dtsi"
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <351 0x0>;
+	qcom,board-id = <8 5>;
+	
+	gta2xlwifi-generic {
+		model = "Samsung Tab A2 XL WIFI Rev05";
+		compatible = "samsung,gta2xlwifi-eur", "qcom,sdm450", "qcom,msm8953", "lk2nd,device";
+		lk2nd,match-bootloader = "T590*";
+		lk2nd,pstore = <0x86100000 0x100000>;
+	};
+	
+	// Bootloader won't continue if it can't delete some nodes from below
+	soc {
```

Comment:
Hi, can you test if it still works without this? It should only be necessary on some motorola phones.

Reply:
@alikates Actually it's other way around, everything except Samsung SDM450 boards probably doesn't need this.